### ISSUE TITLE
How to use pretrained syntactic models in Syntaxnet?

### ISSUE THREAD

**Issue by aluminumbox** [tensorflow/models#4238]
I am still very new to syntaxnet. And I am trying to use the pretrained models, which claim to be Parsey McParseface's cousins.

It seems that parsey_universal is a baseline model. However, in different pretrained models, there are different files, such as char-ngram-map . They all miss context.pbtxt file, which is necessary for the parser/tokenize.

At first, I thought the pretrained model should be ready to use. But if we use parse_universal for all models, how can we reproduce the nice accuracy on Universal Dependencies? Do I have to download the UD dataset to train the model first?

I've been stuck on this for two days. I hope someone can show me the right path, in case I trained the model in vain.

Many thanks!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by karmel** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!

**Comment by coldplaying42** [comment]
@aluminumbox  Have you addressed the problem of missing context.pbtxt file? I face the same problem.

**Comment by aluminumbox** [comment]
> @aluminumbox Have you addressed the problem of missing context.pbtxt file? I face the same problem.

well that is really a long time ago. I remember check out a certain commit and there is a context.pbtxt file which can be used for other languages. I was using it for Chinese parsing and get correct results.
